vllm/entrypoints/cli/openai.py

```diff
@@ -136,6 +145,12 @@ def subparser_init(
             default=None,
             help=("The system prompt to be added to the chat template, "
                   "used for models that support system prompts."))
+        chat_parser.add_argument("-q",
+                                 "--quick-chat",
```

Comment:
```suggestion
                                 "--quick",
```

IMO `chat` (and similarly `complete`) are redundant since that's in the command name already